maven-jahia-plugin/src/main/java/org/jahia/utils/maven/plugin/support/AetherHelperFactory.java

```diff
@@ -75,7 +75,7 @@
     public static AetherHelper create(PlexusContainer container, MavenProject project, MavenSession session, Log log)
             throws MojoExecutionException {
         try {
-            if (container.hasComponent("org.sonatype.aether.RepositorySystem")) {
+            if (container.hasComponent("org.eclipse.aether.RepositorySystem")) {
```

Comment:
This is wrong, because here the check was done to detect the aether implementation of Maven 3.0. Because of that change I now see this warning during the build:
```
[INFO] Using Aether helper for Maven 3.0.x
[WARNING]
[WARNING] ************************* DEPRECATION *************************
[WARNING] *                                                             *
[WARNING] * The version of Maven (3.0.x), you are using, is deprecated. *
[WARNING] * Please, switch to a more recent one (e.g. 3.3.x).           *
[WARNING] *                                                             *
[WARNING] ***************************************************************
```
although I am using Maven 3.3.

I think that it is due time to also include https://jira.jahia.org/browse/TECH-177 for the release of jahia 8.1.0.0. In that case we should simply stop building here instead of trying to continue. 

We should also remove these lines from jahia-parent/pom.xml
```
    <prerequisites>
        <maven>3.0</maven>
    </prerequisites>
```
It is enough to just use the setting in the maven-enforcer-plugin further down in the file, which is already enforcing Maven 3.3.0 or higher.

Reply:
Ok will tackle this in fact, no need to keep all this 3.0, 3.1 code